lib/transport_web/templates/user/organizations.html.eex

```diff
@@ -5,10 +5,10 @@
 <div class="cards">
     <%= for organization <- @organizations do %>
     <div class="card">
-        <div class="organization-image">
+        <div class="image">
```

Comment:
Maybe semantics of this should've been something like ```card-image``` to be clearer about their depth.